cmd/eskip/mediadefaults.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
as the suggested code above should show, too, i think the strategy pattern is a big overkill for this case. we're just setting default output for write commands and default input for read commands, and nothing else.